### ISSUE TITLE
Outsiders can switch repeaters, comparators and maps/item frames

### ISSUE THREAD

**Issue by aco4** [rDankMemes/Minecraft-Server-Support#32]
**Describe the bug**
Players not part of a town (i.e. an outsider) can still change the state of repeaters, comparators, and maps/item frames despite any permission setting. This can break redstone circuitry that requires precise timings, or other cause other unintended effects. Or in the case of maps/item frames, it can just be annoying seeing outsiders rotate them.

**To Reproduce**
1. Go to another town that you are not part of
2. Find a repeater or comparator, and right click it (or item frame/map)
3. See error

**Expected behavior**
The expected behavior is that outsiders should not be able to interfere with redstone, or rotate item frames.

**Screenshots**
none

**Your Environment**
 - Client Version: 1.13.2
 - Linux
 - Litematica 1.13.2

**Additional context**

**Comment by justin-venetucci** [comment]
TL;DR: Towny devs say "hurr durr this isn't possible.  You can just limit the amount of repeaters banners barrels furnaces and stuff you use" which is dum.

**Comment by justin-venetucci** [comment]
We can recommend that people encase their redstone carefully inside plots but we all know that's much easier said than done...
